src/LEGO.AsyncAPI/Models/AsyncApiSerializableExtensions.cs

```diff
@@ -46,14 +47,15 @@ public static void SerializeAsYaml<T>(this T element, Stream stream, AsyncApiVer
         /// <param name="stream">The given stream.</param>
         /// <param name="specVersion">The AsyncApi specification version.</param>
         /// <param name="format">The output format (JSON or YAML).</param>
+        [Obsolete($"Please use overridden version that accepts a {nameof(AsyncApiWriterSettings)} instance.")]
```

Comment:
Is the version with settings actually implemented?

Reply:
Yup it's just below this function, it already existed before

Reply:
While taking a second glance I added overrided versions of the other methods as well